### PR TITLE
Publish documentation

### PR DESCRIPTION
**Pull request by rachelcarmena**
## Pending tasks

* [ ] Create another S3 bucket for Arrow Meta Documentation
* [ ] Configure S3_BUCKET environment variable in the repository
* [ ] Configure AWS secret in the repository

@franciscodr will help with the pending tasks

### CODE REVIEW

**Comment by nomisRev** [comment]
> Access to Bitrise (Bitrise? GitHub Actions can be used for it)
I think moving to GitHub Actions would be preferred, Arrow should also do so later. We added Bitrise when we ran into memory issues with Ank in Arrow on Travis, those have been fixed now but the CI legacy remains.  :(

**Comment by rachelcarmena** [comment]
> > Access to Bitrise (Bitrise? GitHub Actions can be used for it)
> 
> I think moving to GitHub Actions would be preferred, Arrow should probably also do so later. Bitrise was added when we ran into memory issues with Ank in Arrow on Travis, those have been fixed now but the CI legacy remains. :(

Thanks @nomisRev !! We'll change that legacy after testing it here :+1:

**Comment by rachelcarmena** [comment]
> > Access to Bitrise (Bitrise? GitHub Actions can be used for it)
> 
> I think moving to GitHub Actions would be preferred, Arrow should probably also do so later. Bitrise was added when we ran into memory issues with Ank in Arrow on Travis, those have been fixed now but the CI legacy remains. :(

I updated the description and the content of the pull request :wink: Thanks again @nomisRev !!

File: .github/workflows/publish_website.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            echo "All fine, all good with bundler"
+        else
+            echo "Updating bundler"
+            gem uninstall bundler --force
+            gem install bundler --force
+            echo "$( gem list | grep bundler )"
+        fi
+        bundle install --gemfile docs/Gemfile --path vendor/bundle
+        export BUNDLE_GEMFILE=docs/Gemfile bundle exec jekyll serve -s docs
+        export BUNDLE_GEMFILE=modules/docs/arrow-docs/Gemfile
```
Comment (franciscodr) [inline .github/workflows/publish_website.yml]: I think this line is redundant and it's pointing out to a path of the arrow repo, so we could remove it
```suggestion
```

Comment (rachelcarmena) [reply .github/workflows/publish_website.yml]: Thanks @franciscodr! OMG, I think I pushed a WIP version. I fixed more commands: https://github.com/arrow-kt/arrow-meta/pull/50/commits/46e444bfa4ddfac55c7c240317f45de5505e7f08

**Pull request merged**